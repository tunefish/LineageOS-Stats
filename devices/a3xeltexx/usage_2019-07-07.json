{
  "14.1-20170416-unofficial-a3xeltexx": 7,
  "14.1-20170703-unofficial-a3xeltexx": 2,
  "14.1-20170711-unofficial-a3xeltexx": 9,
  "14.1-20170713-unofficial-a3xeltexx": 1,
  "14.1-20170807-unofficial-a3xeltexx": 1,
  "14.1-20170826-unofficial-a3xeltexx": 5,
  "14.1-20170907-unofficial-a3xeltexx": 16,
  "14.1-20170909-unofficial-a3xeltexx": 4,
  "14.1-20171025_105805-unofficial-a3xeltexx": 10,
  "14.1-20171027_160221-unofficial-a3xeltexx": 1,
  "14.1-20171101_122209-unofficial-a3xeltexx": 15,
  "14.1-20171114_052935-unofficial-a3xeltexx": 3,
  "14.1-20171123_112040-unofficial-a3xeltexx": 1,
  "14.1-20171201_185323-unofficial-a3xeltexx": 26,
  "14.1-20180411-unofficial-a3xeltexx": 210,
  "14.1-20190320-unofficial-a3xeltexx": 3,
  "15.1-20180228-unofficial-a3xeltexx": 4,
  "15.1-20180302-unofficial-a3xeltexx": 1,
  "15.1-20180305-unofficial-a3xeltexx": 8,
  "15.1-20180320-unofficial-a3xeltexx": 11,
  "15.1-20180329-unofficial-a3xeltexx": 43,
  "15.1-20180410-unofficial-a3xeltexx": 216,
  "15.1-20180413-unofficial-a3xeltexx": 481,
  "15.1-20180606-unofficial-a3xeltexx": 2,
  "15.1-20180625-unofficial-a3xeltexx": 1,
  "15.1-20180729-unofficial-a3xeltexx": 8,
  "15.1-20180803-unofficial-a3xeltexx": 9,
  "15.1-20180822-unofficial-a3xeltexx": 3,
  "15.1-20180826-unofficial-a3xeltexx": 3,
  "15.1-20180830-unofficial-a3xeltexx": 5,
  "15.1-20180908-unofficial-a3xeltexx": 3,
  "15.1-20180914-unofficial-a3xeltexx": 5,
  "15.1-20180921-unofficial-a3xeltexx": 40,
  "15.1-20180928-unofficial-a3xeltexx": 14,
  "15.1-20181014-unofficial-a3xeltexx": 11,
  "15.1-20181109-unofficial-a3xeltexx": 23,
  "15.1-20181127-unofficial-a3xeltexx": 65,
  "15.1-20190112-unofficial-a3xeltexx": 28
}