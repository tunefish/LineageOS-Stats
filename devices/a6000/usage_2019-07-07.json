{
  "14.1-20170125-unofficial-a6000": 11,
  "14.1-20170205-unofficial-a6000": 55,
  "14.1-20170210-unofficial-a6000": 18,
  "14.1-20170225-unofficial-a6000": 180,
  "14.1-20170228-unofficial-a6000": 5,
  "14.1-20170318-unofficial-a6000": 1,
  "14.1-20170403-unofficial-a6000": 63,
  "14.1-20170422-nightly-a6000": 17,
  "14.1-20170514-unofficial-a6000": 11,
  "14.1-20170607-unofficial-a6000": 9,
  "14.1-20170614-unofficial-a6000": 10,
  "14.1-20170706-unofficial-a6000": 43,
  "14.1-20170706-unofficial-dev_harsh1998-a6000": 7,
  "14.1-20170721-unofficial-a6000": 2,
  "14.1-20170731-unofficial-hyperteam-a6000": 6,
  "14.1-20170813-unofficial-a6000": 12,
  "14.1-20170831-unofficial-a6000": 226,
  "14.1-20180108-official-a6000": 2,
  "14.1-20180108-unofficial-a6000": 12,
  "15.1-20180210-hyper-a6000": 27,
  "15.1-20180210-hyper-rendang": 10,
  "15.1-20180216-hyper-a6000": 132,
  "15.1-20180219-hyper-a6000": 25,
  "15.1-20180228-unofficial-a6000": 1,
  "15.1-20180308-hyper-a6000": 27,
  "15.1-20180413-hyper-a6000": 84,
  "15.1-20180514-unofficial-a6000": 2,
  "15.1-20180604-hyper-a6000": 2,
  "15.1-20180611-hyper-a6000": 13,
  "15.1-20180708-unofficial-a6000": 1,
  "15.1-20180720-hyper-a6000": 8,
  "15.1-20180720-unofficial-a6000": 5,
  "15.1-20180724-hyper-a6000": 77,
  "15.1-20180807-hyper-stable-a6000": 572,
  "15.1-20180807-hyper-stable-rendang": 11,
  "15.1-20190220-unofficial-a6000": 1,
  "15.1-20190504-unofficial-a6000": 2,
  "15.1-20190505-unofficial-a6000": 4,
  "16.0-20190301-unofficial-a6000": 84,
  "16.0-20190302-unofficial-a6000": 209,
  "16.0-20190305-unofficial-a6000": 92,
  "16.0-20190309-hyper-a6000": 144,
  "16.0-20190316-hyper-a6000": 446,
  "16.0-20190316-hyper-a6010": 2,
  "16.0-20190406-unofficial-a6000": 25,
  "16.0-20190407-unofficial-a6000": 157,
  "16.0-20190417-unofficial-a6000": 122,
  "16.0-20190508-unofficial-a6000": 12,
  "16.0-20190509-unofficial-a6000": 152,
  "artificial digital project": 1,
  "nuclearom-v-unofficial[7.1.2]--20171117-2103": 77
}