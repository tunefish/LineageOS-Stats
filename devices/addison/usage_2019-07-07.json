{
  "14.1-20170118-unofficial-addison": 1,
  "14.1-20170309-unofficial-addison": 4,
  "14.1-20170401-unofficial-addison": 19,
  "14.1-20170424-nightly-addison": 1,
  "14.1-20170508-nightly-addison": 5,
  "14.1-20170626-nightly-addison": 1,
  "14.1-20170703-nightly-addison": 1,
  "14.1-20170710-nightly-addison": 1,
  "14.1-20170731-nightly-addison": 1,
  "14.1-20170807-nightly-addison": 1,
  "14.1-20170828-nightly-addison": 1,
  "14.1-20170904-nightly-addison": 2,
  "14.1-20170914-unofficial-addison": 97,
  "14.1-20170918-nightly-addison": 1,
  "14.1-20170925-nightly-addison": 3,
  "14.1-20171002-nightly-addison": 1,
  "14.1-20171023-nightly-addison": 1,
  "14.1-20171106-nightly-addison": 5,
  "14.1-20171202_082735-unofficial-addison": 1,
  "14.1-20171209_190925-unofficial-addison": 1,
  "14.1-20171217-microg-addison": 1,
  "14.1-20180120-unofficial-addison": 1,
  "14.1-20180122-nightly-addison": 1,
  "14.1-20180219-nightly-addison": 1,
  "14.1-20180228-nightly-addison": 1,
  "14.1-20180307-nightly-addison": 1,
  "14.1-20180314-nightly-addison": 1,
  "14.1-20180321-nightly-addison": 1,
  "14.1-20180404-nightly-addison": 1,
  "14.1-20180418-nightly-addison": 2,
  "14.1-20180425-nightly-addison": 3,
  "14.1-20180502-nightly-addison": 3,
  "14.1-20180523-nightly-addison": 2,
  "14.1-20180530-nightly-addison": 2,
  "14.1-20180606-nightly-addison": 4,
  "14.1-20180613-nightly-addison": 4,
  "14.1-20180620-nightly-addison": 1,
  "14.1-20180627-nightly-addison": 1,
  "14.1-20180718-nightly-addison": 2,
  "14.1-20180801-nightly-addison": 3,
  "14.1-20180808-nightly-addison": 3,
  "14.1-20180810-unofficial-addison": 1,
  "14.1-20180815-nightly-addison": 3,
  "14.1-20180822-nightly-addison": 1,
  "14.1-20180829-nightly-addison": 4,
  "14.1-20180905-nightly-addison": 2,
  "14.1-20180912-nightly-addison": 1,
  "14.1-20180919-nightly-addison": 5,
  "14.1-20180926-nightly-addison": 3,
  "14.1-20180927-unofficial-addison": 1,
  "14.1-20181003-nightly-addison": 3,
  "14.1-20181010-nightly-addison": 2,
  "14.1-20181017-nightly-addison": 2,
  "14.1-20181024-nightly-addison": 7,
  "14.1-20181031-nightly-addison": 10,
  "14.1-20181107-nightly-addison": 22,
  "14.1-20181204-nightly-addison": 31,
  "14.1-20190102-microg-addison": 1,
  "14.1-20190104-nightly-addison": 202,
  "14.1-20190123-microg-addison": 1,
  "14.1-20190206-microg-addison": 1,
  "14.1-20190214-microg-addison": 2,
  "14.1-20190221-microg-addison": 1,
  "14.1-20190302-microg-addison": 30,
  "14.1-20190408-unofficial-addison": 1,
  "15.1-20180227-unofficial-addison": 8,
  "15.1-20180717-unofficial-addison": 1,
  "15.1-20180729-unofficial-addison": 1,
  "15.1-20180822-unofficial-addison": 4,
  "15.1-20180909-nightly-addison": 1,
  "15.1-20181013-nightly-addison": 4,
  "15.1-20181125-nightly-addison": 8,
  "15.1-20190113-nightly-addison": 8,
  "15.1-20190202-nightly-addison": 1,
  "15.1-20190203-unofficial-addison": 5,
  "15.1-20190214-nightly-addison": 41,
  "15.1-20190214-unofficial-addison": 23,
  "15.1-20190316-nightly-addison": 123,
  "15.1-20190316-unofficial-addison": 55,
  "15.1-20190318-unofficial-addison": 1,
  "15.1-20190522-nightly-addison": 49,
  "15.1-20190522-unofficial-addison": 18,
  "15.1-20190529-unofficial-addison": 1,
  "16.0-20181227-unofficial-addison": 55,
  "16.0-20190109-unofficial-addison": 62,
  "16.0-20190210-unofficial-addison": 195,
  "16.0-20190311-unofficial-addison": 311,
  "16.0-20190409-unofficial-addison": 246,
  "16.0-20190510-unofficial-addison": 229,
  "8.1.0-addison": 4
}