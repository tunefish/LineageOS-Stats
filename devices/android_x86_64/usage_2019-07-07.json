{
  "14.1-20170228-unofficial-android_x86_64": 6,
  "14.1-20171031_162750-unofficial-android_x86_64": 2,
  "14.1-20180821-unofficial-android_x86_64": 5,
  "14.1-20190128-unofficial-android_x86_64": 1,
  "14.1-20190311-unofficial-android_x86_64": 1,
  "14.1-20190415-unofficial-android_x86_64": 2,
  "14.1-20190430-unofficial-android_x86_64": 1,
  "14.1.0-android_x86_64": 4276,
  "swat2015-lineaeos14.1": 1
}