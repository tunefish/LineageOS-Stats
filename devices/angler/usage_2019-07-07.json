{
  "13.0-20170216-unofficial-angler": 1,
  "13.0-20170925-unofficial-angler": 1,
  "13.0-20180716-unofficial-angler": 11,
  "14.1-20170123-unofficial-angler": 3,
  "14.1-20170127-unofficial-angler": 14,
  "14.1-20170204-unofficial-angler": 4,
  "14.1-20170212-nightly-angler": 2,
  "14.1-20170215-unofficial-angler": 10,
  "14.1-20170303-unofficial-angler": 6,
  "14.1-20170313-nightly-angler": 54,
  "14.1-20170320-nightly-angler": 3,
  "14.1-20170326-unofficial-angler": 4,
  "14.1-20170403-nightly-angler": 1,
  "14.1-20170410-nightly-angler": 2,
  "14.1-20170424-nightly-angler": 2,
  "14.1-20170522-nightly-angler": 2,
  "14.1-20170605-nightly-angler": 4,
  "14.1-20170612-nightly-angler": 1,
  "14.1-20170617-unofficial-angler": 14,
  "14.1-20170626-nightly-angler": 1,
  "14.1-20170704-unofficial-angler": 1,
  "14.1-20170705-unofficial-angler": 3,
  "14.1-20170710-nightly-angler": 4,
  "14.1-20170717-nightly-angler": 62,
  "14.1-20170724-nightly-angler": 10,
  "14.1-20170731-nightly-angler": 1,
  "14.1-20170807-nightly-angler": 1,
  "14.1-20170814-nightly-angler": 4,
  "14.1-20170821-nightly-angler": 3,
  "14.1-20170904-nightly-angler": 4,
  "14.1-20170911-nightly-angler": 3,
  "14.1-20170918-nightly-angler": 1,
  "14.1-20170925-nightly-angler": 1,
  "14.1-20171002-nightly-angler": 5,
  "14.1-20171009-nightly-angler": 15,
  "14.1-20171016-nightly-angler": 6,
  "14.1-20171023-nightly-angler": 12,
  "14.1-20171106-nightly-angler": 2,
  "14.1-20171113-nightly-angler": 3,
  "14.1-20171120-nightly-angler": 4,
  "14.1-20171127-nightly-angler": 3,
  "14.1-20171204-nightly-angler": 2,
  "14.1-20171211-nightly-angler": 1,
  "14.1-20171218-nightly-angler": 2,
  "14.1-20171225-nightly-angler": 1,
  "14.1-20180101-nightly-angler": 2,
  "14.1-20180108-nightly-angler": 4,
  "14.1-20180122-nightly-angler": 5,
  "14.1-20180129-nightly-angler": 4,
  "14.1-20180205-nightly-angler": 3,
  "14.1-20180212-nightly-angler": 1,
  "14.1-20180219-microg-angler": 6,
  "14.1-20180219-nightly-angler": 45,
  "14.1-20180625-unofficial-angler": 1,
  "14.1-20180826-unofficial-angler": 3,
  "14.1-20190314-unofficial-angler": 2,
  "14.1-20190401-unofficial-angler": 1,
  "14.1-20190415-unofficial-angler": 1,
  "14.1-20190501-unofficial-angler": 12,
  "14.1-20190502-unofficial-angler": 1,
  "14.1-20190531-unofficial-angler": 2,
  "14.1-20190603-unofficial-angler": 2,
  "15.1-20180205-unofficial-angler": 2,
  "15.1-20180208-unofficial-angler": 5,
  "15.1-20180215-unofficial-angler": 3,
  "15.1-20180220-unofficial-angler": 13,
  "15.1-20180301-nightly-angler": 3,
  "15.1-20180312-nightly-angler": 5,
  "15.1-20180319-nightly-angler": 5,
  "15.1-20180326-nightly-angler": 10,
  "15.1-20180409-nightly-angler": 6,
  "15.1-20180415-nightly-angler": 1,
  "15.1-20180416-nightly-angler": 9,
  "15.1-20180423-nightly-angler": 11,
  "15.1-20180430-nightly-angler": 7,
  "15.1-20180507-nightly-angler": 9,
  "15.1-20180514-nightly-angler": 13,
  "15.1-20180521-nightly-angler": 7,
  "15.1-20180528-nightly-angler": 9,
  "15.1-20180605-nightly-angler": 5,
  "15.1-20180611-nightly-angler": 3,
  "15.1-20180618-nightly-angler": 8,
  "15.1-20180625-nightly-angler": 8,
  "15.1-20180702-nightly-angler": 6,
  "15.1-20180709-nightly-angler": 10,
  "15.1-20180716-nightly-angler": 7,
  "15.1-20180723-nightly-angler": 15,
  "15.1-20180730-nightly-angler": 8,
  "15.1-20180806-nightly-angler": 11,
  "15.1-20180813-nightly-angler": 13,
  "15.1-20180820-nightly-angler": 11,
  "15.1-20180827-nightly-angler": 9,
  "15.1-20180903-microg-angler": 1,
  "15.1-20180903-nightly-angler": 14,
  "15.1-20180910-microg-angler": 1,
  "15.1-20180910-nightly-angler": 11,
  "15.1-20180917-nightly-angler": 11,
  "15.1-20180924-nightly-angler": 13,
  "15.1-20181001-nightly-angler": 14,
  "15.1-20181008-nightly-angler": 16,
  "15.1-20181015-nightly-angler": 23,
  "15.1-20181022-microg-angler": 1,
  "15.1-20181022-nightly-angler": 12,
  "15.1-20181029-nightly-angler": 11,
  "15.1-20181101-unofficial-angler": 30,
  "15.1-20181105-nightly-angler": 17,
  "15.1-20181112-nightly-angler": 8,
  "15.1-20181113-nightly-angler": 1,
  "15.1-20181114-nightly-angler": 10,
  "15.1-20181115-nightly-angler": 4,
  "15.1-20181116-nightly-angler": 4,
  "15.1-20181117-nightly-angler": 6,
  "15.1-20181118-nightly-angler": 2,
  "15.1-20181119-microg-angler": 1,
  "15.1-20181119-nightly-angler": 9,
  "15.1-20181120-nightly-angler": 3,
  "15.1-20181121-nightly-angler": 2,
  "15.1-20181122-nightly-angler": 1,
  "15.1-20181123-nightly-angler": 3,
  "15.1-20181124-nightly-angler": 7,
  "15.1-20181125-nightly-angler": 2,
  "15.1-20181126-microg-angler": 1,
  "15.1-20181126-nightly-angler": 2,
  "15.1-20181127-nightly-angler": 2,
  "15.1-20181128-nightly-angler": 3,
  "15.1-20181129-nightly-angler": 3,
  "15.1-20181130-nightly-angler": 4,
  "15.1-20181201-nightly-angler": 3,
  "15.1-20181202-nightly-angler": 7,
  "15.1-20181203-nightly-angler": 3,
  "15.1-20181204-nightly-angler": 3,
  "15.1-20181205-nightly-angler": 2,
  "15.1-20181206-nightly-angler": 1,
  "15.1-20181207-nightly-angler": 3,
  "15.1-20181208-nightly-angler": 17,
  "15.1-20181208-unofficial-angler": 1,
  "15.1-20181209-nightly-angler": 3,
  "15.1-20181210-nightly-angler": 3,
  "15.1-20181212-nightly-angler": 1,
  "15.1-20181212-unofficial-angler": 1,
  "15.1-20181213-nightly-angler": 2,
  "15.1-20181214-nightly-angler": 6,
  "15.1-20181214-unofficial-angler": 1,
  "15.1-20181215-nightly-angler": 7,
  "15.1-20181216-nightly-angler": 1,
  "15.1-20181217-nightly-angler": 1,
  "15.1-20181218-nightly-angler": 3,
  "15.1-20181219-nightly-angler": 2,
  "15.1-20181220-nightly-angler": 6,
  "15.1-20181221-nightly-angler": 2,
  "15.1-20181222-nightly-angler": 2,
  "15.1-20181223-nightly-angler": 4,
  "15.1-20181224-nightly-angler": 2,
  "15.1-20181225-nightly-angler": 1,
  "15.1-20181226-nightly-angler": 4,
  "15.1-20181227-nightly-angler": 1,
  "15.1-20181228-nightly-angler": 3,
  "15.1-20181229-nightly-angler": 2,
  "15.1-20181230-nightly-angler": 3,
  "15.1-20181231-nightly-angler": 7,
  "15.1-20190101-nightly-angler": 2,
  "15.1-20190102-nightly-angler": 8,
  "15.1-20190103-nightly-angler": 4,
  "15.1-20190104-nightly-angler": 18,
  "15.1-20190105-nightly-angler": 11,
  "15.1-20190106-nightly-angler": 12,
  "15.1-20190107-nightly-angler": 7,
  "15.1-20190108-nightly-angler": 9,
  "15.1-20190109-nightly-angler": 2,
  "15.1-20190110-nightly-angler": 2,
  "15.1-20190111-nightly-angler": 3,
  "15.1-20190112-nightly-angler": 7,
  "15.1-20190113-nightly-angler": 9,
  "15.1-20190114-microg-angler": 2,
  "15.1-20190114-nightly-angler": 6,
  "15.1-20190115-nightly-angler": 7,
  "15.1-20190116-nightly-angler": 5,
  "15.1-20190117-nightly-angler": 4,
  "15.1-20190118-nightly-angler": 5,
  "15.1-20190119-nightly-angler": 4,
  "15.1-20190120-nightly-angler": 4,
  "15.1-20190121-nightly-angler": 4,
  "15.1-20190122-nightly-angler": 2,
  "15.1-20190123-nightly-angler": 4,
  "15.1-20190124-nightly-angler": 8,
  "15.1-20190126-nightly-angler": 11,
  "15.1-20190127-nightly-angler": 10,
  "15.1-20190128-nightly-angler": 4,
  "15.1-20190129-nightly-angler": 10,
  "15.1-20190130-nightly-angler": 4,
  "15.1-20190131-nightly-angler": 2,
  "15.1-20190201-nightly-angler": 5,
  "15.1-20190202-nightly-angler": 6,
  "15.1-20190203-nightly-angler": 9,
  "15.1-20190204-nightly-angler": 5,
  "15.1-20190205-nightly-angler": 12,
  "15.1-20190206-nightly-angler": 8,
  "15.1-20190208-nightly-angler": 9,
  "15.1-20190209-nightly-angler": 11,
  "15.1-20190210-nightly-angler": 7,
  "15.1-20190211-nightly-angler": 14,
  "15.1-20190212-nightly-angler": 8,
  "15.1-20190213-nightly-angler": 2,
  "15.1-20190214-nightly-angler": 12,
  "15.1-20190215-nightly-angler": 8,
  "15.1-20190216-nightly-angler": 9,
  "15.1-20190217-nightly-angler": 16,
  "15.1-20190218-microg-angler": 3,
  "15.1-20190218-nightly-angler": 7,
  "15.1-20190219-nightly-angler": 6,
  "15.1-20190220-nightly-angler": 6,
  "15.1-20190221-nightly-angler": 9,
  "15.1-20190222-nightly-angler": 11,
  "15.1-20190222-unofficial-angler": 4,
  "15.1-20190223-nightly-angler": 8,
  "15.1-20190224-nightly-angler": 49,
  "15.1-20190225-microg-angler": 7,
  "15.1-20190301-nightly-angler": 188,
  "15.1-20190306-unofficial-angler": 1,
  "15.1-20190307-unofficial-angler": 1,
  "15.1-20190308-microg-angler": 7,
  "15.1-20190308-nightly-angler": 260,
  "15.1-20190310-unofficial-angler": 2,
  "15.1-20190311-microg-angler": 12,
  "15.1-20190311-unofficial-angler": 2,
  "15.1-20190312-unofficial-angler": 6,
  "15.1-20190315-nightly-angler": 291,
  "15.1-20190316-unofficial-angler": 1,
  "15.1-20190317-unofficial-angler": 2,
  "15.1-20190319-microg-angler": 3,
  "15.1-20190320-unofficial-angler": 3,
  "15.1-20190321-unofficial-angler": 1,
  "15.1-20190322-nightly-angler": 461,
  "15.1-20190326-microg-angler": 10,
  "15.1-20190326-unofficial-angler": 2,
  "15.1-20190327-unofficial-angler": 1,
  "15.1-20190330-unofficial-angler": 1,
  "15.1-20190402-microg-angler": 17,
  "15.1-20190405-nightly-angler": 210,
  "15.1-20190407-unofficial-angler": 1,
  "15.1-20190409-microg-angler": 8,
  "15.1-20190412-nightly-angler": 252,
  "15.1-20190414-unofficial-angler": 1,
  "15.1-20190416-microg-angler": 6,
  "15.1-20190419-nightly-angler": 202,
  "15.1-20190423-microg-angler": 11,
  "15.1-20190426-nightly-angler": 530,
  "15.1-20190430-microg-angler": 5,
  "15.1-20190507-microg-angler": 5,
  "15.1-20190514-microg-angler": 13,
  "15.1-20190514-unofficial-angler": 3,
  "15.1-20190515-unofficial-angler": 1,
  "15.1-20190517-nightly-angler": 399,
  "15.1-20190520-unofficial-angler": 4,
  "15.1-20190521-unofficial-angler": 2,
  "15.1-20190527-unofficial-angler": 3,
  "15.1-20190529-unofficial-angler": 1,
  "15.1-20190531-nightly-angler": 134,
  "15.1-20190603-unofficial-angler": 2,
  "15.1-20190604-unofficial-angler": 5,
  "15.1-20190605-unofficial-angler": 7,
  "16.0-20181001-unofficial-angler": 423,
  "16.0-20190120-unofficial-angler": 4,
  "16.0-20190209-unofficial-angler": 1,
  "16.0-20190311-unofficial-angler": 44,
  "16.0-20190408-unofficial-angler": 34,
  "16.0-20190511-unofficial-angler": 43,
  "lineageos 16.0 self-built cgmod": 1,
  "mos-7.1.2-rc1.0-angler": 1,
  "mos-7.1.2-rc3.0-angler": 1,
  "mos-7.1.2-stable3.0-angler": 9,
  "njh47d": 6,
  "to_angler-userdebug 7.1.2 njh47f 80f29644fb test-keys": 1
}