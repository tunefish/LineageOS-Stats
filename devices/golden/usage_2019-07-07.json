{
  "13.0-20170118-unofficial-golden": 5,
  "13.0-20170121-unofficial-golden": 8,
  "13.0-20170129-unofficial-golden": 107,
  "13.0-20170224-unofficial-golden": 8,
  "13.0-20170302-unofficial-golden": 19,
  "13.0-20170310-unofficial-golden": 105,
  "13.0-20170328-unofficial-golden": 843,
  "13.0-20170420-unofficial-golden": 14,
  "13.0-20170905-unofficial-glee": 1,
  "13.0-20180315-unofficial-golden": 2,
  "13.0-20180620-unofficial-golden": 1,
  "14.1-20170122-unofficial-golden": 205,
  "14.1-20170318-unofficial-golden": 3,
  "14.1-20170402-unofficial-golden": 12,
  "14.1-20170415-unofficial-golden": 5,
  "14.1-20170419-unofficial-golden": 11,
  "14.1-20170503-unofficial-golden": 3,
  "14.1-20170507-unofficial-golden": 40,
  "14.1-20170516-unofficial-golden": 3,
  "14.1-20170522-unofficial-golden": 10,
  "14.1-20170609-unofficial-golden": 766,
  "14.1-20170610-unofficial-golden": 15,
  "14.1-20170628-unofficial-golden": 958,
  "14.1-20170724-unofficial-golden": 54,
  "14.1-20170817-unofficial-golden": 73,
  "14.1-20170920-unofficial-golden": 17,
  "14.1-20171011-unofficial-golden": 155,
  "14.1-20171215_192311-official-golden": 65,
  "14.1-20171215_192311-unofficial-golden": 1517,
  "14.1-20180109_002536-unofficial-golden": 418,
  "14.1-20181014-unofficial-golden": 2,
  "15.1-20181030-unofficial-golden": 7
}